controllers/movieController.js

```diff
@@ -0,0 +1,41 @@
+const { Movie } = require('../models');
+const { configs } = require('../utils');
+const parser = require('../utils/parser');
+
+/**
+ * Returns all data from every movie.
+ * @return  array of movies
```

Comment:
Add params to the JDoc comment, since you have a request and response. Also the type of the return, for readability.

```suggestion
/**
 * Returns all data from every movie.
 * @param {??} req
 * @param {??} res
 * @return {Promise} array of movies
 */
```